### ISSUE TITLE
add --cleanup-aws option to remove *ANY* AWS allocated resource

### ISSUE THREAD

**Issue by gserra-olx** [goadapp/goad#157]
in particular:
 - cloudwatch logs 
 - SQS queues

**Comment by cwaltken-edrans** [comment]
Aggreed 👍

**Comment by VitorFalcao** [comment]
@cwaltken-edrans Are you already working on a fix? I would like to start contributing to the project and this issue seems interesting.

**Comment by cwaltken-edrans** [comment]
No, I'm not working on this one right now. Feel free to send a PR.
I'm pretty sure we can get it merged.

**Comment by VitorFalcao** [comment]
Thank you. Could you please give me some pointers, since I am new to the project? What other resources should the "--cleanup-aws" option delete?

**Comment by cwaltken-edrans** [comment]
Please take a look at: https://github.com/cwaltken-edrans/goad/blob/master/infrastructure/aws/aws.go
The code in this file is solely responsible for creating aws resources.

Resources to delete if present that come right to my mind would be:
 * SQS-Queue
 * Role for lambda function
 * Lambda aliases
 * Lambda functions
 * CloudWatch logs associated to the lambda function

Ideally the cleanup code would also apply to the Docker infrastructure though to run goad on docker is still considered to be an experimental feature.
You could take a look at the cleanup code for each infrastructure and improve on that and reuse the code.

**Comment by thalesfsp** [comment]
Any update on it?

**Comment by cwaltken-edrans** [comment]
Right now, no one is working on this.
